nz_address/address.go

```diff
@@ -10,7 +10,7 @@ import (
 type Address struct {
 	UnitType        string `json:"unit_type"`
 	UnitIdentifier  string `json:"unit_identifier"`
-	StreetNumber    int    `json:"street_number"`
+	StreetNumber    string `json:"street_number"`
```

Comment:
This should remain an `int`, and there should be another one added - `StreetNumberHigh`. That's the way GeoPAF + others do it.